uefi-test-runner/src/main.rs

```diff
@@ -28,6 +28,7 @@ fn efi_main(image: Handle, mut st: SystemTable<Boot>) -> Status {
     let mut buf = String::new();
     st.firmware_vendor().as_str_in_buf(&mut buf).unwrap();
     info!("Firmware Vendor: {}", buf.as_str());
+    assert_eq!(buf, "EDK II");
```

Comment:
nit: this whole section could be refactored to

```rust
    info!("Firmware Vendor: {}", st.firmware_vendor());
    assert!(st.firmware_vendor().eq_str_until_nul(&"EDK II"));
```

Reply:
Simplified a bit. I kept the `assert_eq` since that has the advantage of printing the actual value if it fails, but switched to using `to_string()`. (Back when this code was written, there were no "host unit tests", so this test runner was also testing things like the string conversion functions, but no need to keep that now.)

Reply:
alright